third_party/amd/lib/TritonAMDGPUTransforms/AccelerateAMDMatmul.cpp

```diff
@@ -1254,17 +1251,16 @@ class TritonAMDGPUAccelerateMatmulPass
     switch (auto isaFamily = triton::AMD::deduceISAFamily(archGenerationName)) {
     case ISAFamily::CDNA1:
     case ISAFamily::CDNA2:
-    case ISAFamily::CDNA3: {
-      patterns.add<::ScaledBlockedToScaledMFMAF8F6F4>(
-          context, getMfmaVersion(isaFamily, archGenerationName),
-          matrixInstructionSize,
-          /*benefit=*/10);
+    case ISAFamily::CDNA3:
       patterns.add<::BlockedToMFMA, ::ScaledBlockedToMFMA>(
-          context, getMfmaVersion(isaFamily, archGenerationName),
-          matrixInstructionSize, kPack,
+          context, getMfmaVersion(isaFamily), matrixInstructionSize, kPack,
+          /*benefit=*/2);
+      break;
+    case ISAFamily::CDNA4:
```

Comment:
This would cause we don't have `BlockedToMFMA` pattern for CDNA4? I think we need to also add it here.